profile/README.md

```diff
@@ -0,0 +1,54 @@
+# We are NewAperio
+
+![hero images](https://user-images.githubusercontent.com/1529366/173092313-2938e23d-8e4b-4d93-87f8-c827d38c4002.png)
+
+### What we do:
+
+We’re an agile, fully distributed team of developers and designers who sweat the details and take pride in our craft. We design and build web and mobile apps for clients in:
+- Healthcare 🩺 🏥 💊
+- Fitness 👟🥦🏋🏽
+- Education 🏫 ✏️🎓
+- Civic engagement 🗳️🪧📣
+- Developer tools 🕹️🛠️🤖
+
+In addition to our work in these core industries, we have completed numerous projects in other domains such as an app for conducting remote property inspections, an app that makes it easier to manage receipts and track expenses, and an app that turns codebases into visual maps.
+
+We also provide staff augmentation, so if your team is looking for developers or designers with particular technical expertise, or for expanded capacity during a busy period, let’s connect.
+
+### We are experts in:
+- Elixir
+- Phoenix
+- JavaScript
+- TypeScript
+- React
+- React Native
+- GraphQL
+- Postgres
+- Product Design Sprints
+- UI Design 
+- UX Design
+- Design Systems
+
+### Our history and values:
+
+Logan Leger founded NewAperio in 2010 while a student at Louisiana State University. The company has grown steadily since to include multiple developers and designers prototyping, designing, and building apps for clients nationwide. 
+
+| Learning & Mentorship 📖 | Generosity 🎪 |
+| :--- | :--- |
+| We value curiosity and life-long learning. We strive for continuous improvement and invest in each other's growth. | We give back to our community. Recently, that meant direct aid to people adversely affected by Hurricane Ida. |
+
+| Flexibility 🤸🏾 | Humanity 🧑🏼‍🤝‍🧑🏻 |
+| :--- | :--- |
+| We approach problems with humility. Resilience and agility allow us to pivot quickly when something isn't working. | We value empathy, equity, and inclusion. When making decisions, people are our first priority. |
+
+| Craft 🪡 | Trust 🧠 |
+| :--- | :--- |
+| Designing and building software should improve lives and reduce harm for all users, contributors, and affected people. | We hire highly skilled people and we trust their competence. We give people autonomy to do their best work.
+
+### Let's connect
+
+Interested in working with our team on a project? Schedule a meeting with our founder, [Logan here](https://savvycal.com/logan/chat). 
```

Comment:
```suggestion
Interested in working with our team on a project? Schedule a meeting with our founder [Logan here](https://savvycal.com/logan/chat). 
```

Not sure how best to edit this but I think the comma is erroneous. I think removing it, vs adding another one, reads better.